pyproject.toml

```diff
@@ -8,6 +8,7 @@ authors = [{name = "Maren Buettner", email = "maren.buettner@helmholtz-muenchen.
 readme = "README.md"
 dynamic = ["version"]
 description = "Pytometry is a Python package for flow and mass cytometry analysis."
+requires-python = '>= 3.9'
```

Comment:
Python 3.9 is the lowest python version that is still supported by numpy (see "NEP29") and is also used across scverse packages. 
Having Python >= 3.8 as a minimum requirement enables use of the `Literal` type.